EIPS/payoutable-token-draft.md

```diff
@@ -0,0 +1,302 @@
+---
+eip: <to be assigned>
+title: Payoutable Token
+author: Fernando Paris <fer@io.builders>
+status: Draft
+type: Standards Track
+category:  ERC
+created: 2019-05-10
+requires: 20
+
+---
+
+## Simple Summary
+An extension to the ERC-20 standard token that allows Token wallet owners to request payut from their wallet, by calling the smart contract and attaching a payout instruction string.
+
+## Actors
+
+#### Token Wallet Owners
+The person or company who owns the wallet, and will order payout.
+
+#### Token contract operator
+The entity, company responsible/owner of the token contract, and token issuing/minting. This actor is in charge of trying to fullfill all payout request(s), reading the payout instruction(s), and corelate the payout details.
```

Comment:
fullfill mut be fulfill,
corelate must be correlate

In the other EIPs I have used `operator` as an account, who can do things on behalf of another account. So for example in Clearable Token an operator can order a clearable transfer for another account. This is in accordance with ERC-777 (as far as I understand it). So maybe instead of using operator here we could use agent or something